src/main/resources/application.yml

```diff
@@ -59,6 +17,113 @@ management:
     servlet:
       context-path: /_monitoring
 
```

Comment:
instead of duplicating the whole base config, the default value can be passed as
scheduler-service.synchronizer.type=couchbase.

as spring boot app.yml parameters can be overridden with environment variables, you can pass SCHEDULER_SERVICE_SYNCHRONIZER_TYPE=redis where you deploy your project